src/ucp/core/ucp_request.c

```diff
@@ -290,8 +290,11 @@ ucs_status_t ucp_request_test(void *request, ucp_tag_recv_info_t *info)
     ucp_request_t *req = (ucp_request_t*)request - 1;
 
     if (req->flags & UCP_REQUEST_FLAG_COMPLETED) {
+        /* NOTE: workaround for gtest */
         if (req->flags & UCP_REQUEST_FLAG_RECV) {
-            *info = req->recv.info;
+            *info = req->recv.tag.info;
+        } else if (req->flags & UCP_REQUEST_FLAG_STREAM_RECV) {
```

Comment:
where is this flag set?

Reply:
In next PR with `ucp_stream_recv_nb` implementation

Reply:
then pls move to next PR